mne/viz/ica.py

```diff
@@ -350,7 +350,7 @@ def set_title_and_labels(ax, title, xlab, ylab):
 
         # the actual plot
         fig = _plot_ica_properties(
-            pick, ica, inst, psds_mean, freqs, len(picks),
+            pick, ica, inst, psds_mean, freqs, ica_data.shape[1],
```

Comment:
If that controls the y axis scale shouldn't it be `inst._data.shape[0]` or `_segment_raw(inst, ...).shape[0]` (the number of epochs)?

Reply:
At that time point, ica_data is:

`ica_data = np.swapaxes(epochs_src.get_data()[:, picks, :], 0, 1)`

Reply:
ok!